### ISSUE TITLE
Unsupported download strategy `{:path=>"."}`.

### ISSUE THREAD

**Issue by Mactub07** [adamski/podspec-generator#3]
when I create podspec.file and then run "pod install" I've got an error "Unsupported download strategy `{:path=>"."}`."

see code 
sh script podspec-generator node_modules/react-native-navbar --ios=9.0 

content of generated file 

Pod::Spec.new do |s|
  s.name         = "ReactNativeNavbar"
  s.version      = "2.1.0"
  s.summary      = "Simple customizable navbar component for react-native"
  s.homepage     = "https://github.com/react-native-community/react-native-navbar"
  s.license      = { :type => "MIT" }
  s.authors      = { "Alexey Kureev" => "kureev-mail@ya.ru" }
  s.platform     = :ios, "9.0"
  s.source       = { :path => "." }
  s.source_files = "ios", "ios/**/*.{h,m}"
end

**Comment by adamski** [comment]
Please post your `package.json`

You should be able to remove that line, or replace it with something like 
```
s.source         = { :git => 'https://your-repository-url', :tag => s.version }
```
Example output from one of my modules:
```
s.source         = { :git => package['repository']['url'], :tag => s.version }

```

**Comment by Mactub07** [comment]
{
  "_from": "react-native-navbar@^2.1.0",
  "_id": "react-native-navbar@2.1.0",
  "_inBundle": false,
  "_integrity": "sha512-hvETP5lgBbokmTHNMuHdlG6LRqDyXGJb8eiTuxmZ0RSHTsDbGBkxrSBdS9PIA+dizej8Ql9tBQFf6e0QDNG/WA==",
  "_location": "/react-native-navbar",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "react-native-navbar@^2.1.0",
    "name": "react-native-navbar",
    "escapedName": "react-native-navbar",
    "rawSpec": "^2.1.0",
    "saveSpec": null,
    "fetchSpec": "^2.1.0"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/react-native-navbar/-/react-native-navbar-2.1.0.tgz",
  "_shasum": "4f21e947501ce5cf4563a29914cb3192db35a647",
  "_spec": "react-native-navbar@^2.1.0",
  "_where": "/Users/dennis/WebstormProjects/yonoton-app-sdk",
  "author": {
    "name": "Alexey Kureev",
    "email": "kureev-mail@ya.ru",
    "url": "https://github.com/Kureev"
  },
  "bugs": {
    "url": "https://github.com/react-native-community/react-native-navbar/issues"
  },
  "bundleDependencies": false,
  "dependencies": {
    "prop-types": "^15.5.10"
  },
  "deprecated": false,
  "description": "Simple customizable navbar component for react-native",
  "devDependencies": {
    "babel-eslint": "^7.1.1",
    "eslint": "^3.16.1",
    "eslint-config-airbnb": "^14.1.0",
    "eslint-plugin-import": "^2.2.0",
    "eslint-plugin-jsx-a11y": "^4.0.0",
    "eslint-plugin-react": "^6.10.0"
  },
  "homepage": "https://github.com/react-native-community/react-native-navbar",
  "keywords": [
    "react-native",
    "navbar"
  ],
  "license": "MIT",
  "main": "index.js",
  "name": "react-native-navbar",
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/react-native-community/react-native-navbar.git"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "version": "2.1.0"
}

**Comment by Mactub07** [comment]
it's not my module, I've just wanted to create podspec for modules, which have no podspec file

**Comment by adamski** [comment]
Yes thats the reason I created the module. I don't have time to look at it now, but if you want to take a look at the code and send a PR that would be great.

**Comment by adamski** [comment]
Just been looking at this again - the line `s.source = { :path => "." }` is generated by default. All my generated podspecs have this and I don't get an error. 

What version of Cocoapods are you using? Do you still have the issue?

**Comment by Mactub07** [comment]
I dont able to check pod version now, but I remember that latest version of Pod dont support this one

**Comment by adamski** [comment]
I just updated to CocoaPods 1.4 and was able to install. An example output podspec:
```ruby
Pod::Spec.new do |s|
  s.name         = "ReactNativeAutogrowTextinput"
  s.version      = "4.1.0"
  s.summary      = "React Native auto-growing multiline text input"
  s.homepage     = "https://github.com/wix/react-native-autogrow-textinput"
  s.license      = { :type => "MIT" }
  s.authors      = { "Artal Druk" => "artald@wix.com" }
  s.platform     = :ios, "8.0"
  s.source       = { :path => "." }
  s.source_files = "ios", "ios/**/*.{h,m}"
end

```
